data/json/items/generic.json

```diff
@@ -109,6 +109,20 @@
     "volume": 1,
     "flags": [ "TRADER_AVOID" ]
   },
+  {
+    "type": "GENERIC",
+    "id": "fp_loyalty_card",
+    "category": "other",
+    "symbol": "-",
+    "color": "pink",
+    "name": "foodplace loyalty card",
+    "description": "A bright pink loyalty card,  all the points are stamped.  This would definitly prove your fidelity to Foodplace, if it still meant anything...",
```

Comment:
Extraneous space after a comma, `definitly` is misspelled and should be `definitely`.